Code/Framework/AzCore/AzCore/Debug/Profiler.inl

```diff
@@ -10,44 +10,48 @@
 
 namespace AZ::Debug
 {
+    namespace Platform
```

Comment:
Is `Platform` the right namespace name to use here?

Reply:
Since these functions are branching off into platform specific code, it's more or less accurate.  I guess it or the functions could be updated to reflect they are associated to the Profiler and not "global" `AZ::Debug` namespace if this seems too generic.

Reply:
What do you think about `Profiler` instead?

Reply:
Changing to `Profiler` works

Reply:
Looks like the `Platform` namespace is sticking around here, unless you have another suggestion that isn't `Profiler` (it conflicts with the `Profiler` interface, forgot about that).